acceptance/acceptance_test.go

```diff
@@ -211,7 +211,21 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 				t.Parallel()
 			}
 
-			runTest(t, dir, coverDir, repls.Clone(), config, configPath)
+			expanded := internal.ExpandEnvMatrix(config.EnvMatrix)
```

Comment:
If we don't allow an empty env list we should be able to simplify this and only include the else block here?

Also can you make the control flow easier to read, i.e. make it clear the default case and the special case when `config.EnvMatrix` is configured. Right now we rely on the predicate `len(expanded) == 1 && len(expanded[0]) == 0` to check for the default case when `EnvMatrix` is not configured.